File: src/main/kotlin/stellarwitch7/ram/spell/trick/blunder/RAMNotSupportedBlunder.kt
package stellarwitch7.ram.spell.trick.blunder

import dev.enjarai.trickster.spell.trick.Trick
import dev.enjarai.trickster.spell.trick.blunder.TrickBlunderException
import net.minecraft.text.MutableText

class RAMNotSupportedBlunder(source: Trick) : TrickBlunderException(source) {
    override fun createMessage(): MutableText {
        return super.createMessage().append("Caster does not support RAM")
    }
}

File: src/main/scala/stellarwitch7/ram/spell/blunder/RAMNotSupportedBlunder.scala
package stellarwitch7.ram.spell.blunder

import dev.enjarai.trickster.spell.trick.Trick
import dev.enjarai.trickster.spell.blunder.TrickBlunderException
import net.minecraft.text.MutableText

class RAMNotSupportedBlunder(source: Trick) extends TrickBlunderException(source) {
  override def createMessage(): MutableText =
    super.createMessage().append("Caster does not support RAM")
}
